script/get-release-notes.js

```diff
@@ -0,0 +1,27 @@
+import fs from "node:fs";
+import path from "node:path";
+import process from "node:process";
+
+const manifestFile = path.resolve("./package.json");
+const changelogFile = path.resolve("./CHANGELOG.md");
+
+const manifestRaw = fs.readFileSync(manifestFile).toString();
+const manifest = JSON.parse(manifestRaw);
+const version = manifest.version;
```

Comment:
There's some duplication of [`bump-changelog.js`](https://github.com/ericcornelissen/svgo-action/blob/a25150ce676edb6c2d9516245ed5814f4df0793d/script/bump-changelog.js) in this file which would ideally be resolved before this is merged.